training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -74,12 +74,10 @@
     "%%time\n",
```

Comment:
<p>For <code>train_gpt_simple.py</code>,change:</p><p>the Pytorch estimator -&gt; the HuggingFace estimator </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>